lib/src/map/widget.dart

```diff
@@ -104,10 +101,10 @@ class _FlutterMapStateContainer extends State<FlutterMap>
             child: ColoredBox(color: widget.options.backgroundColor),
           ),
           ...widget.children.map(
-            (child) => TranslucentPointer(
-              translucent: widget.options.applyPointerTranslucencyToLayers,
-              child: child,
-            ),
+            // ignore: deprecated_member_use_from_same_package
+            (child) => widget.options.applyPointerTranslucencyToLayers
+                ? TranslucentPointer(child: child)
+                : child,
```

Comment:
If `applyPointerTranslucencyToLayers` isn't used, we don't have to map all children.

Reply:
Whoops, my oversight. Fixed.